dev/integrations/ml/databricks/README.md

```diff
@@ -1,4 +1,4 @@
-# Use Intel Optimized ML libraries in Databricks Runtime for ML
```

Comment:
in Databricks Runtime for ML => on azure databricks cloud with Databricks Runtime 7.5 ML version